<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPri</class>
 <widget class="QMainWindow" name="MenuPri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTitulo">
      <property name="text">
       <string>Social Structure - Bismarck Romero 201708880</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnIniciarSesion">
      <property name="text">
       <string>Iniciar Sesion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRegistrarse">
      <property name="text">
       <string>Registrarse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnInformacion">
      <property name="text">
       <string>Informacion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSalir">
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelCorreo">
      <property name="text">
       <string>Correo:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputCorreo"/>
    </item>
    <item>
     <widget class="QLabel" name="labelContrasenia">
      <property name="text">
       <string>Contrase√±a:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputContrasenia"/>
    </item>
    <item>
     <widget class="QLabel" name="labelMensaje"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
